core/src/main/java/nl/nn/adapterframework/webcontrol/api/AuthorizationFilter.java

```diff
@@ -27,55 +27,56 @@
 import javax.ws.rs.Produces;
 import javax.ws.rs.container.ContainerRequestContext;
 import javax.ws.rs.container.ContainerRequestFilter;
+import javax.ws.rs.container.ResourceInfo;
+import javax.ws.rs.core.Context;
 import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
 import javax.ws.rs.core.SecurityContext;
 import javax.ws.rs.ext.Provider;
 
-import nl.nn.adapterframework.util.LogUtil;
-
 import org.apache.log4j.Logger;
-import org.jboss.resteasy.core.Headers;
-import org.jboss.resteasy.core.ResourceMethodInvoker;
-import org.jboss.resteasy.core.ServerResponse;
+
+import nl.nn.adapterframework.util.LogUtil;
 
 /**
- * Manages authorization per resource/collection.
+ * Manages authorization per resource/collection. / it was based on nl.nn.adapterframework.webcontrol.api.AuthorizationFilter
  * 
  * @since	7.0-B1
- * @author	Niels Meijer
+ * @author	Carlo Camiletti
```

Comment:
It's okey if you want to put your name here, but it's bad practice to remove other peoples names.
You can have multiple @author annotations